extras/vagrantExample/src/main/vagrant/Vagrantfile

```diff
@@ -52,8 +52,8 @@ Vagrant.configure(2) do |config|
     export ACCUMULO_VERSION=1.6.5
     ###export ACCUMULO_VERSION=1.7.1
     export HADOOP_VERSION=2.7.2
-    export RYA_EXAMPLE_VERSION=3.2.10-SNAPSHOT
-    export SESAME_VERSION=2.7.6
+    export RYA_EXAMPLE_VERSION=3.2.13-SNAPSHOT
```

Comment:
This will need to be 4.0.0. This commit should be after the version update commit.

Reply:
Changed